dist/index.html

```diff
@@ -5,16 +5,21 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
     <title>To do list</title>
-<script defer src="main.js"></script></head>
+<script defer src="/to-do-list/main.js"></script></head>
```

Comment:
- [ ] On the first instance I tried to run the project locally but functions were not working then I noticed this. It is working perfectly as `<script defer src="main.js"></script>`. Kindly try to update to the correct path.